index.js

```diff
@@ -41,7 +41,7 @@ function parse(selector) {
     }
   }
 
-  if (className.length) {
+  if (className.length) { // eslint-disable-line unicorn/explicit-length-check
```

Comment:
we can add >0, but... 2 more bytes for nothing serious. Really don't like such rules. It will be 0, so, falsey value, so won't pass the check, so not make sense.